tasks/collections/README.md

```diff
@@ -13,4 +13,4 @@ gulp = install(gulp);
 
 // tasks are now injected and ready to be used
 gulp.start('install');
-```
\ No newline at end of file
+```
```

Comment:
There is no change at all it seems?

Reply:
![image](https://user-images.githubusercontent.com/2228672/166159226-192e1266-f9ed-4134-b129-72a42445a0bd.png)

there is missing EOL too, for some reasons, GH does not render 
![image](https://user-images.githubusercontent.com/2228672/166159246-2de561c9-e44c-431e-b129-f1a47ccc6bf3.png)
symbol on the left side of the diff